src/Aura/Http/Adapter/Stream.php

```diff
@@ -314,6 +314,11 @@ protected function setContext()
 
         // what scheme are we using?
         $url = parse_url($this->request->url);
+
+        if (empty($url['scheme'])) {
+            $url['scheme'] = 'http';
```

Comment:
That is correct as that's the safest, and most widely used standard, way of defaulting the value. I basically followed all browsers and how curl does it now internally.